src/saved_model.rs

```diff
@@ -124,6 +124,16 @@ impl TensorInfo {
     pub fn name(&self) -> &OutputName {
         &self.name
     }
+    
+    /// Returns the data type of the tensor.
+    pub fn dtype(&self) -> &DataType {
```

Comment:
Since `DataType` is trivial to copy (i.e. is small and implements `Copy`), we can probably go ahead and just return it by value.

Reply:
Whoops. Good idea. Done.